apps/gallery/stories/components/wui-icon-box.stories.ts

```diff
@@ -0,0 +1,57 @@
+import type { Meta } from '@storybook/web-components'
+import { clockSvg } from '@web3modal/ui/src/assets/svg/clock'
```

Comment:
I think a separate story is needed to display all svg's in a sort of list / grid. Then for icon components let's just use 2 as preview. Updating these whenever you add more icons in each component will be too big of a chore. Same can be applied to other stories like icon component